apps/src/sites/hourofcode.com/pages/views/swiper_page_beyond.js

```diff
@@ -0,0 +1,65 @@
+// Adds carousel functionality to hourofcode.com/beyond
+
+// This installs the Swiper library WebComponent.
+// See docs here: https://swiperjs.com/swiper-api#web-component.
+
+// import function to register Swiper custom elements
+import {register} from 'swiper/element/bundle';
+// register Swiper custom elements
+register();
+
+const actionBlocksParams = {
```

Comment:
Do these change from page to page? Wondering if it's worth pulling this out into a shared file?

Reply:
I've thought about this, and refactoring how we can set up Swiper to be less repeat-y in general 🙌  Wanted to wait until everything was moved over and cleaned up though. I'll make a ticket for the backlog for this.

Reply:
thanks for making the ticket!